html/js/product-search.js

```diff
@@ -379,9 +379,17 @@ function display_product_summary(target, product) {
 		}
 
 		// check if the product attribute has an associated knowledge panel that exists
-		if ((attribute.panel_id) && (document.getElementById("panel_" + attribute.panel_id))) {
-			// onclick : open the panel content + reflow to make sur all column content is shown
-			card_html = '<a href="#panel_' + attribute.panel_id + '" onclick="document.getElementById(\'panel_' + attribute.panel_id + '_content\').classList.add(\'active\'); $(document).foundation(\'equalizer\', \'reflow\');"' + card_html + '</a>';
+		if (attribute.panel_id) {
+			// note: on the website, the id for the panel contains : instead of - (e.g. for the ingredients_analysis_en:vegan panel)
+			var panel_element_id = 'panel_' + attribute.panel_id.replace(':', '-');
+			if (document.getElementById(panel_element_id)) {
+				// onclick : open the panel content + reflow to make sur all column content is shown			
+				card_html = '<a href="#' + panel_element_id
+				+ '" onclick="document.getElementById(\'' + panel_element_id + '_content\').classList.add(\'active\'); $(document).foundation(\'equalizer\', \'reflow\');"' + card_html + '</a>';
```

Comment:
I'm not sure that on the new design reflow will work, so this may have to be tested.

Reply:
We currently need the reflow because we use the equalizer function of Foundation (for setting columns of equal height). We don't have this anymore on the new design, so I think it will work. (and we'll be able to remove all the reflow calls)

Reply:
:+1: